histonets_cv/cli.py

```diff
@@ -571,5 +572,42 @@ def graph(image, regions, pathfinding_method, simplification_method,
     return edges_to_graph(edges, fmt=format)
 
 
+@main.command()
+@click.option('-d', '--dilation', type=click.IntRange(0, 100),
+              default=1,
+              help='Dilation radius to thicken the binarized image prior to '
+                   'perform skeletonization. '
+                   'Ranges from 0 to 100. Defaults to 6.')
+@click.option('-p', '--passes', type=click.IntRange(0, 100),
```

Comment:
Should this be 'iterations' to match the parameter within the function, just for consistency?